Essentials/src/com/earth2me/essentials/signs/SignEnchant.java

```diff
@@ -24,24 +24,24 @@ protected boolean onSignCreate(final ISign sign, final User player, final String
         try {
             stack = sign.getLine(1).equals("*") || sign.getLine(1).equalsIgnoreCase("any") ? null : getItemStack(sign.getLine(1), 1, ess);
         } catch (SignException e) {
-            sign.setLine(1, "§c<item|any>");
+            sign.setLine(1, ChatColor.translateAlternateColorCodes('&', ess.getSettings().getErrorSignColor("<item|any>")));
```

Comment:
No such method `ess.getSettings().getErrorSignColor()` ?

Reply:
It's just an example of getting into the config.